src/system/mod.rs

```diff
@@ -106,7 +108,8 @@ impl FileCloser {
 
 fn close_range(min_fd: c_uint, max_fd: c_uint) -> io::Result<()> {
     if min_fd <= max_fd {
-        cerr(unsafe { libc::syscall(libc::SYS_close_range, min_fd, max_fd, 0 as c_uint) })?;
+        // SAFETY: this function is safe to call with these arguments
```

Comment:
I'd go a bit further and mention that this function it is always safe to call because:
- Any errors while closing a specific fd will be effectively ignored.
- If the range is invald (min_fd > max_fd) that's an error but not UB.
- If the flags are invalid that's also an error but not UB.